R/validate_model_data.R

```diff
@@ -196,13 +196,16 @@ validate_model_data <- function(hub_path, file_path, round_id_col = NULL,
     )
     if (is_any_error(checks$spl_compound_taskid_set)) {
       return(checks)
+    } else {
+      compound_taskid_set <- checks$spl_compound_taskid_set$compound_taskid_set
```

Comment:
We first validate and detect the compound task id set and if it validates, we pass the compound task id set to subsequent tests.